byoqm/visuals/dashboard.py

```diff
@@ -13,15 +15,30 @@ def __init__(self, start_date: datetime, end_date: datetime):
         self._start_date = start_date
         self._end_date = end_date
 
-    def show_graphs(self):
+    def _check_data(self, filepath, in_use_qm, target_path):
+        with open(filepath) as f:
+            reader = csv.reader(f)
+            for row in reader:
+                if row[0] == "qualitymodel" and row[1] != in_use_qm:
+                    return False
+                if row[0] == "src_root" and ("./" + row[1]) != target_path:
+                    return False
+        return True
```

Comment:
I think naming would be clearer `_check_data` is split into two methods: `_check_src_root` and `_check_quality_model`.

Reply:
👍